docker/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+set -eu
+
+# Compiler configuration
+
+. submodules/beaver/lib/dlang.sh
+
+set_dc_dver
+
+# Install dependencies
+
+case "$DMD" in
+    dmd*   ) PKG= ;;
+    1.*    ) PKG="dmd1=$DMD-$DIST" ;;
+    2.*.s* ) PKG="dmd-transitional=$DMD-$DIST" ;;
+    2.*    ) if [ $(echo $DMD | cut -d. -f2) -ge 077 ]; then
+                PKG="dmd-compiler=$DMD dmd-tools=$DMD libphobos2-dev=$DMD"
+             else
+                PKG="dmd-bin=$DMD libphobos2-dev=$DMD"
+             fi ;;
+    *      ) echo "Unknown \$DMD ($DMD)" >&2; exit 1 ;;
+esac
```

Comment:
Can we not use beaver for this?

Reply:
Nope, beaver does not exposes this functionality as a library shell script - https://github.com/sociomantic-tsunami/beaver/blob/v0.x.x/bin/dlang/install#L13

Reply:
:+1: Thanks!